### PR TITLE
CAL-409 Renamed unit test files

### PR DESCRIPTION
**Pull request by kcover**
#### What does this PR do?

Renames a test file conform with standards.

#### Who is reviewing it? 

@vinamartin @mcalcote @ahoffer

#### Choose 2 committers to review/merge the PR.

@clockard
@rzwiefel

#### How should this be tested?

Run the test files

#### What are the relevant tickets?

[CAL - 409] (https://codice.atlassian.net/browse/CAL-409)

#### Checklist:
- [ ] Documentation Updated
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/TreUtilityTest.java [hunk 1]
```diff
@@ -55,8 +55,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class TreTestUtility {
-  private static final Logger LOGGER = LoggerFactory.getLogger(TreTestUtility.class);
+public class TreUtilityTest {
```
Comment (mcalcote) [inline catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/TreUtilityTest.java]: Unfortunate that we have a utility with tests in it.

**Comment by kcover** [comment]
build now

**Comment by vinamartin** [comment]
build now

File: catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/TreUtilityTest.java (hunk 1 above)
Comment (clockard) [reply catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/TreUtilityTest.java]: That is unfortunate

**Pull request merged**